upup/models/cloudup/resources/addons/node-problem-detector.addons.k8s.io/k8s-1.17.yaml.template

```diff
@@ -1,6 +1,59 @@
 {{ with .NodeProblemDetector }}
 # Sourced from https://github.com/kubernetes/node-problem-detector/tree/v0.8.8
 ---
+# We need service account and role binding for node-problem-detector
+# see: https://github.com/kubernetes/node-problem-detector/issues/149 for details
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: node-problem-detector
+  namespace: kube-system
+  labels:
+    app: node-problem-detector
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: node-problem-detector
+  labels:
+    app: node-problem-detector
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - nodes/status
+  verbs:
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+  - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: node-problem-detector-binding
+  labels:
+    app: node-problem-detector
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: node-problem-detector
```

Comment:
Why not use the existing `system:node-problem-detector` ClusterRole instead of defining our own here?

Reply:
I didn't aware the system default clusterRole, Thanks for your advice.